### DISCUSSION TITLE
PaddleOCR/ppstructure/overview

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13924] [Q&A]
# PaddleOCR/ppstructure/overview

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/ppstructure/overview.html

**Comment by lanhong123** [comment]
为什么我使用的效果非常差，基本可以认为是在胡乱识别，是哪里出问题了吗，大家能达到官网说的准确率吗‘’
